airflow/timetables/interval.py

```diff
@@ -58,6 +58,10 @@ def _align(self, current: DateTime) -> DateTime:
         """
         raise NotImplementedError()
 
+    def _align_to_prev(self, current: DateTime) -> DateTime:
+        """Align given time to the scheduled."""
```

Comment:
Please

1. Add more description on what this function does (similar to the docstring for `_align_to_next`)
2. Describe _why_ this is different from `self._prev(self._align_to_next(t))` (when a DAG’s schedule changes, as you mentioned in the original PR).